core/domain/suggestion_registry.py

```diff
@@ -599,8 +599,8 @@ def pre_accept_validate(self):
             self.change.state_name, self.change.content_id)
         if content_html != self.change.content_html:
             raise utils.ValidationError(
-                'The given content_html does not match the content of the '
-                'exploration.')
+                'The Exploration content has changed since this translation '
```

Comment:
I don't actually understand why this would be an issue, as long as the reviewer can see the *current* exploration content while reviewing. We could have this as a note in the frontend (I would strongly recommend this, together with guidance to "check it carefully") but it probably shouldn't block submission? Otherwise if someone fixes a typo in the original content then it will invalidate all translations suggested for it.

Reply:
Per discussion [here](https://docs.google.com/document/d/15iaCn7XEchKNgAVZ0Nn1-gyg42HL9k86n-DR_WLQR_s/edit?disco=AAAALh9sfTk), we will be going ahead with this approach of rejecting the suggestion if it fails the "pre_accept_validate" check.

Created https://github.com/oppia/oppia/issues/12013 to make sure that review screen is updated with the latest original content.

Reply:
For the follow-up issue could you please include "revisit the decision of whether we need the content check to be in pre_accept_validate" and the rationale for considering that?

Reply:
Added!